.github/workflows/develop.yml

```diff
@@ -263,3 +263,22 @@ jobs:
           flags: rust
           fail_ci_if_error: false
           verbose: true
+
+  compat:
+    name: Compatibility Test
+    if: github.event.pull_request.draft == false
+    needs: build
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v3
+      - name: Download pre-built binaries
+        uses: actions/download-artifact@v4
+        with:
+          name: bins
+          path: .
+      - name: Unzip binaries
+        run: |
+          mkdir -p ./bins/current
+          tar -xvf ./bins.tar.gz --strip-components=1 -C ./bins/current
+      - run: ./tests/compat/test-compat.sh 0.6.0
```

Comment:
I wonder if it's possible to auto inference this `old_ver` based on current version and compatibility requirement rules. So that we don't have to add another step during release.

Reply:
I think it's not needed to change this `old_ver` across releases. We test backward and forward compat at the same time, so the compat has some kind of ”transitivity“: if version 0.7.0 and 0.8.0 are both compat with version 0.6.0, then version 0.8.0 is compat with 0.7.0. I'd like to keep things simple for now. Besides, there's no place to find the latest release version consider the "latest" in github's release doesnot necessary mean the latest(or highest, newest) release.